source/collaborate/install-desktop-app.rst

```diff
@@ -168,7 +168,11 @@ You can download the `desktop app <https://mattermost.com/apps/>`_ directly from
 
 .. tab:: Generic Linux
 
-  A AppImage distribution (Beta) of a compressed tarball is available. Automatic app updates are supported and enabled. When a new version of the desktop app is released, your app updates automatically. 
+  The Desktop app is available in two formats which are usable on most Linux distributions: a compressed tarball, and an AppImage binary. Both can be downloaded from the `Desktop App's Github releases page <https://github.com/mattermost/desktop/releases>`_. Automatic app updates are supported and enabled. When a new version of the desktop app is released, your app updates automatically.
```

Comment:
@mvitale1989 @cwarnermm A point of clarity: Automatic updates are only supported on the AppImage build, not the `.tar.gz`

Reply:
Thanks, @devinbinnie - I'll clarify in a follow-on docs PR